src/test/java/streambms/sender/BatteryManagementSystemSenderTest.java

```diff
@@ -0,0 +1,26 @@
+package streambms.sender;
+
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertNull;
+
+import org.junit.Test;
+
+import streambms.model.Battery;
+
+public class BatteryManagementSystemSenderTest {
```

Comment:
Focus on testing your functionality - the most crucial functionality of the Sender is the output format.
In the absence of a test for the output format, any future mistakes will be caught only while integrating with the receiver.